### PR TITLE
Add basic metadata to the yaml.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Adds some basic metadata to the OAS.

Further details on metadata to be added to an API are [here](https://github.com/teamdigitale/api-openapi-samples/blob/master/openapi-v3/api-example.yaml).

@LucaPrete @mccalv

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+  x-summary: >-
+    Public Administriations onboarding APIs for the IO project.
+  description: |-
+      The backend used by the onboarding portal for public administrations of the IO project.
+
+      The website for the project is [io-website].
+      Privacy Policy is [privacy].
+
+      Development happens on github.
+
```
Comment (LucaPrete) [inline openapi.yml]: I'd remove a blank line here

File: openapi.yml [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+      [//]:  # (Riferimenti e url)
```
Comment (LucaPrete) [inline openapi.yml]: I would write it in English

File: openapi.yml [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
+      [io-website]: https://io.italia.it
+      [tos]: https://io.italia.it/note-legali/
+      [privacy]: https://io.italia.it/privacy-policy/
+
   version: "0.0.1"
+  x-api-id: 49b844f5-2b1a-47a9-880a-ca480dac3736
```
Comment (LucaPrete) [inline openapi.yml]: Not sure what is this. Is it supposed to be hardcoded?

File: openapi.yml [hunk 4]
```diff
@@ -1,8 +1,31 @@
...
+  contact:
+    url: https://io.italia.it
+    email: teamdigitale@governo.it
+  termsOfService: https://io.italia.it/note-legali/
+externalDocs:
+  url: https://io.italia.it
+  description: IO Project website
 servers:
   - url: https://api.pa-onboarding.dev.io.italia.it/
```
Comment (LucaPrete) [inline openapi.yml]: I know this was here also before, but I'm not sure it should be hardcoded, as opposed to be taken from an env var. This depends by which environment the code is deployed into. @alexgpeppe, any thought?

File: openapi.yml (hunk 3 above)
Comment (ioggstream) [reply openapi.yml]: Just a random UUID in case info.title changes.

File: openapi.yml (hunk 4 above)
Comment (ioggstream) [reply openapi.yml]: @LucaPrete in a public api this should be hardcoded and versioning increased as server changes.

File: openapi.yml (hunk 1 above)
Comment (ioggstream) [reply openapi.yml]: I just made it for you to complete with all the required informations. As I don't know them
I put there some example informations. Edit this PR and I'll provide you feedback.

File: openapi.yml (hunk 2 above)
Comment (ioggstream) [reply openapi.yml]: Like above, it's just a placeholder to show what you can do with `description`. Feel free to use or remove that section. Examples of description are:

- https://developers.italia.it/it/api/siope.html
- https://developers.italia.it/it/api/muoversi-in-lombardia

@LucaPrete @alexgpeppe

Comment (alexgpeppe) [reply openapi.yml]: I don't know if such section is needed... Maybe @mccalv or @paomico can say something about it?

File: openapi.yml [hunk 5]
```diff
@@ -1,8 +1,31 @@
...
 info:
   title: "IO onboarding PA API"
-  description: "The backend used by the onboarding portal for public administrations of the IO project."
+  x-summary: >-
+    Public Administriations onboarding APIs for the IO project.
+  description: |-
+      The backend used by the onboarding portal for public administrations of the IO project.
+
+      The website for the project is [io-website].
+      Privacy Policy is [privacy].
```
Comment (alexgpeppe) [inline openapi.yml]: @mccalv and @paomico, are there or will there be some webpages that can be linked here?

**Pull request merged**